### PR TITLE
UX enhancements

### PR DESCRIPTION
**Pull request by amirfefer**
- [x] Replace the  slider with number input for instances count input
- [x] Change strings, replace `provisioning` with `launch`
- [x] align form fields, radio boxes
- [x] uppercase `aws` in the progress page during errors

### CODE REVIEW

File: src/Components/InstanceCounter/index.js [hunk 1]
```diff
@@ -1,42 +1,55 @@
...
+      input = MIN_INSTANCES;
     }
+    setWizardContext((prevState) => ({
+      ...prevState,
+      chosenNumOfInstances: input,
+    }));
+  };
+
+  const onPlus = () => {
+    const newValue = chosenNumOfInstances + 1;
```
Comment (ezr-ondrej) [inline src/Components/InstanceCounter/index.js:34]: Shall we cap it on MAX_INSTANCES here?

File: src/Components/ProvisioningWizard/steps/Pubkeys/NewKeyForm.js [hunk 2]
```diff
@@ -96,7 +96,7 @@ const NewSSHKeyForm = ({ setStepValidated }) => {
     return regex.test(ssh);
   };
   return (
-    <>
+    <FormGroup isStack>
```
Comment (ezr-ondrej) [inline src/Components/ProvisioningWizard/steps/Pubkeys/NewKeyForm.js:99]: Is it a good idea to nest FormGroups?

File: src/Components/InstanceCounter/index.js [hunk 3]
```diff
@@ -1,42 +1,55 @@
...
-      } else if (inputValue < MIN_INSTANCES) {
-        newValue = MIN_INSTANCES;
-        setLocalInputValue(MIN_INSTANCES);
-      } else {
-        newValue = Math.floor(inputValue);
-      }
+  const [{ chosenNumOfInstances }, setWizardContext] = useWizardContext();
+
+  const onMinus = () => {
+    const newValue = chosenNumOfInstances - 1;
```
Comment (ezr-ondrej) [inline src/Components/InstanceCounter/index.js:12]: Shall we cap on `MIN_INSTANCES`?

**Review by ezr-ondrej — COMMENTED** [review]
Just few nitpicks, in general looks good :+1:

File: src/Components/InstanceCounter/index.js (hunk 1 above)
Comment (amirfefer) [reply src/Components/InstanceCounter/index.js:34]: no need, the `max` prop do it automatically :)

File: src/Components/InstanceCounter/index.js (hunk 3 above)
Comment (amirfefer) [reply src/Components/InstanceCounter/index.js:12]: ditto

File: src/Components/ProvisioningWizard/steps/Pubkeys/NewKeyForm.js (hunk 2 above)
Comment (amirfefer) [reply src/Components/ProvisioningWizard/steps/Pubkeys/NewKeyForm.js:99]: with this we gain the wanted padding between form inputs with no  hardcoding css, I haven't seen any warnings or errors 🤔

Comment (ezr-ondrej) [reply src/Components/ProvisioningWizard/steps/Pubkeys/NewKeyForm.js:99]: :shrug: ok, it just looked weird to me, but if it works, then it works :)

**Review by ezr-ondrej — APPROVED** [review]
Nice, thanks @amirfefer :+1:

**Pull request merged**